hal/serial_api.h

```diff
@@ -211,6 +211,7 @@ void serial_break_clear(serial_t *obj);
  */
 void serial_pinout_tx(PinName tx);
 
+#if DEVICE_SERIAL_FC
```

Comment:
I think that this commit should be removed for now. 
It will probably cause build failure at least here:
https://github.com/ARMmbed/mbed-os/blob/f0ec856020290a2309d713ae8d9b1d51cab7fbe3/targets/TARGET_RDA/TARGET_UNO_91H/serial_api.c#L131-L133

Reply:
Yeah, the build failed as expected. This won't fail after https://github.com/ARMmbed/mbed-os/pull/10994/commits/4842c87815c64401b92a3871b8f79e8f559384a2. Could you re-review, please?